main.go

```diff
@@ -226,10 +226,6 @@ func (o *Options) Validate() error {
 		return fmt.Errorf("must specify --database-dsn with --load-database")
 	}
 
-	if o.DBOnlyMode && o.DSN == "" {
-		return fmt.Errorf("must specify --database-dsn with --db-only-mode")
-	}
```

Comment:
What do you think about having an error if DBOnlyMode is false?

Reply:
Yeah that should work, and still possible to edit the manifests as we'll be defaulting to true per the above recommendation.